### PR TITLE
Version 0.3.0

### LINKED ISSUE #20: Update Gulp / Package.json file
- [x] Update outdated modules
- [x] Remove image optimalisation from flow - Update: we are keeping this.
- [x] Add SVG only optimalisation - Update: we are keeping this.

### LINKED ISSUE #28: Drop Bower for NPM 5 for installing front-end packages

### PR DESCRIPTION
**Pull request by jmslbam**
- [x] Drop Bower - Fixes #28 
- [x] Overwrite BrowserSync url - Fixes #30 
- [x] Update NPM packages - Fixes #20 
- [x] BrowserSync refresh not working on Twig files. By @NielsdeBlaauw 
- [x] Adding a new LESS include in the main LESS file doesn't trigger a new Watch refresh in vagrant. By @NielsdeBlaauw  (extracted this issue out of this milestones) Never was working in the 0.2.0 branch.
- [x] Switch write to manifest lines
- [x] Only optimize images when using --production 
- [ ] Write documentation about NPM only installation and scripts loading
## Disable ImageOptim
Zelf je afbeeldingen van te voren door ImageOptim halen en niet de boel laten gulpen via je vagrant levert veel meer snelheidswinst op.

### CODE REVIEW

**Comment by jmslbam** [comment]
@NielsdeBlaauw  fixed the refreshing issue via https://github.com/BrowserSync/browser-sync/issues/318#issuecomment-98355661

File: INSTALLATION.md [hunk 1]
```diff
@@ -7,72 +7,77 @@ Clarkson Theme is a modern set of tools to take WordPress theme developement bac
...
+If you don't have a `vendor` directory in your theme then follow the next steps:
 1. Install [composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) / [short info](https://www.abeautifulsite.net/installing-composer-on-os-x)
 2. Run `composer install` in this theme folder.
 
-### Install gulp and Bower
+### Install Dependencies
 
-Building the theme requires [node.js](http://nodejs.org/download/). We recommend you update to the latest version of npm: `npm install -g npm@latest`.
-
-From the command line in the `/development` directory:
```
Comment (NielsdeBlaauw) [inline INSTALLATION.md]: Still from /development, right?

File: INSTALLATION.md [hunk 2]
```diff
@@ -7,72 +7,77 @@ Clarkson Theme is a modern set of tools to take WordPress theme developement bac
 | Prerequisite    | How to check | How to install
 | --------------- | ------------ | ------------- |
 | PHP >= 5.4.x    | `php -v`     | [php.net](http://php.net/manual/en/install.php) |
-| Node.js 0.12.x  | `node -v`    | [nodejs.org](http://nodejs.org/) |
-| gulp >= 3.8.10  | `gulp -v`    | `npm install -g gulp` |
-| Bower >= 1.3.12 | `bower -v`   | `npm install -g bower` |
-| Clarkson Core   |              | [github.com](https://github.com/level-level/Clarkson-Core) |
+| Node.js 5.0.x   | `node -v`    | [nodejs.org](http://nodejs.org/) |
+| Clarkson Core 2.x  |              | [github.com](https://github.com/level-level/Clarkson-Core) |
```
Comment (NielsdeBlaauw) [inline INSTALLATION.md]: CC 0.2.x

File: INSTALLATION.md [hunk 3]
```diff
@@ -7,72 +7,77 @@ Clarkson Theme is a modern set of tools to take WordPress theme developement bac
...
+      "browsersyncUrl": "http://localhost:8888/project-name/"
   }
 ...
 ```
 
+#### Custom browsersyncUrl
+If you have a different development url from the rest of your team then create a file `source/browsersync.json` with the following config:
+```
+{
+    "url": "http://local.wordpress.dev"
```
Comment (NielsdeBlaauw) [inline INSTALLATION.md]: .example domain for examples

File: README.md [hunk 4]
```diff
@@ -12,8 +12,13 @@ If you would like to read about the documentation, just check out the [Clarkson]
 
 ## What do we use
 
+### Required
 - Clarkson Core [(A WordPress Twig plugin)](http://wp-clarkson.com/core)
+
+### Optional
+Although you don't need to use these parts, it makes the developement proces a bit smoother.
+
 - Clarkson Hooks [WordPress Hooks and Autoloading Namespaced Classes](https://github.com/level-level/clarkson-hooks)
```
Comment (NielsdeBlaauw) [inline README.md]: Is not optional if it is a require in composer.json

File: INSTALLATION.md (hunk 1 above)
Comment (jmslbam) [reply INSTALLATION.md]: yupp

File: INSTALLATION.md (hunk 3 above)
Comment (jmslbam) [reply INSTALLATION.md]: thanks, changed this

File: README.md (hunk 4 above)
Comment (jmslbam) [reply README.md]: ok

**Pull request merged**